lang/php.md

```diff
@@ -0,0 +1,83 @@
+- Symfony
+- CakePHP
+
+## Setting up PHP environment locally:
+
+Instead of downloading various components separately, I feel it’s good to install all in one packages like [XAMPP]( https://www.apachefriends.org/index.html), [WAMP]( https://www.wampserver.com/en/) which contain Apache, PHP, MySQL, and many other dependencies in a single installation file.
+
+In this tutorial, I’ll guide how to install XAMPP. Before getting started, let’s know the full form of XAMPP
+X — Cross-platform (X), as it supports multiple operating systems
+A — Apache HTTP Server
+M — MariaDB (formerly MySQL)
+P — PHP
+P — PERL
+
+Additionally, XAMPP comes with other useful tools such as Mercury mail server, phpMyAdmin for database administration, Webalizer for web server log analysis, OpenSSL, Apache Tomcat for running Java code, and FileZilla FTP server.
+Currently, the latest stable version of PHP is 7.4
+
+**Step 1:**
+Navigate to https://www.apachefriends.org/download.html and download the latest stable version of PHP according to your OS (windows/Linux/OSX)
+
+**Step 2:**
+Once the download is complete, run the installer file which you downloaded. Follow the instructions mentioned and click next. Select components you want install and choose specific location for installation. Make sure to check the “Private networks, such as my home or work network” option. This is very important. XAMPP won’t work if you don’t check this and click Allow access. Once done, click finish.
+Complete the setup and run XAMPP Control Panel.
+
+**Step 3:**
+In the control panel, click on config button. Click on the start button beside Apache and MySQL modules. You can access your Apache server’s dashboard by clicking on the admin button beside it. Alternatively, you can also reach it via http://localhost/dashboard/ URL in your browser.
+You can access your phpMyAdmin dashboard by clicking the admin button beside MySQL module. Or you can simply go to http://localhost/phpmyadmin/ in your browser. Here, you can manage the MariaDB (or MySQL) databases of your web projects.
+
+**Writing your first hello world program in php:**
+Create a new folder hello in C:\xampp\htdocs\ (The C: drive may vary based on the location you chose for installation). Inside the new folder, right click and create a new file ‘hello.php’. Write the below code in your new file.
+
+```php
+<?php
+echo "Hello World!";
+?>
+```
+
+Navigate to  http://localhost/hello/hello.php in your preferred browser.
+
+If it displays “Hello World!”, you’ve successfully setup XAMPP server and wrote your first few lines in PHP.
+
+> *Congratulations on making it till the end!*
+
+If you’re looking for some projects in PHP for inspiration or contribution, checkout the below projects:
+
+- [Project Celadon](https://github.com/Webwiznitr/Project-celadon-2.0)
+- [Project Sirius](https://github.com/CYBORG-NIT-ROURKELA/project-sirius)
+- [Top open source projects in PHP]( https://github.com/trending/php)
+
+# PHP is case insensitive!
```

Comment:
Remove this heading and this point anywhere after line 37 (while setting up the dev environment.